packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -0,0 +1,67 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "controls": {
+        "gap": {
+          "value": "var(--base-size-16)"
+        },
+        "padding": {
+          "value": "var(--base-size-16) var(--base-size-24)"
+        },
+        "focus": {
+          "boxShadow": {
+            "value": "0 0 0 2px var(--brand-color-focus)"
+          }
```

Comment:
I don't think we need tokens for these, as they aren't the kind of thing we'd want users to customise. They also add some overhead that we can avoid by using the values directly in the stylesheet. 

E.g. What is the advantage a user has in changing the focus presentation here?

Reply:
I was thinking that depending on the branding around the video player they might have customized this for different events. I am happy to remove this, however.

Reply:
I have paired down the tokens in the [SHA](https://github.com/primer/brand/pull/373/commits/7dfa6209762e0ebe9416a9487e55bf0f77f8eac8)

Reply:
Oh interesting. Thanks for clarifying.

- Gap, padding I don't think would be event-specific values 🤔 
- focus colors are globally blue in other components, and shouldn't be customisable for a11y reasons AFAIK. Even if it were, all other focus colors in other colors would still be blue, so this would create visual inconsistencies.

On that basis, I feel we should remove these.